QueryEngine/ArrowResultSet.cpp

```diff
@@ -68,36 +69,36 @@ std::vector<TargetValue> ArrowResultSet::getNextRow(const bool translate_strings
     const auto& column_typeinfo = getColType(i);
     switch (column_typeinfo.get_type()) {
       case kSMALLINT: {
-        CHECK(dynamic_cast<const arrow::NumericArray<arrow::Int16Type>*>(&column));
-        const auto& i16_column = static_cast<const arrow::NumericArray<arrow::Int16Type>&>(column);
```

Comment:
I do not recommend using `arrow::NumericArray` -- use the typedefs

Reply:
Nice, I didn't notice them when I wrote this.